code/datums/traits/negative.dm

```diff
@@ -414,30 +400,6 @@
 	the round. You are not an antagonist, and the rules will treat you the same as other crewmembers.</span>")
 
 /datum/quirk/social_anxiety
```

Comment:
Is there a reason you left the empty definition?

Reply:
i honestly forgot about these

Reply:
reason for removal?

Reply:
nvm i see, i feel like this should be a quirk still. some people want to roleplay this, same as depression